dependencies.yaml

```diff
@@ -72,6 +72,8 @@ dependencies:
           - libraft-distance=23.02.*
           - libraft-nn=23.02.*
           - rmm=23.02.*
+          - c-compiler
+          - cxx-compiler
```

Comment:
Ideally we are trying to keep these lists alphabetized, but that's not a hard requirement.

Reply:
I love keeping things in alphabetical order :)